.github/workflows/build.yml

```diff
@@ -468,8 +469,6 @@ jobs:
           chmod uog+x build/moonbeam
           cd moonbeam-types-bundle
           npm install
-          rm -fr dist
```

Comment:
I'm a bit confused, why are we adding `npm run build` to the previous step and removing `node_modules/.bin/tsc` on this one. 
Is it needed to build it or not ?

Reply:
I though we agreed that it wasn't required so I removed it everywhere but one place. That way we know if the build throws errors